src/cpp/rtps/security/SecurityManager.h

```diff
@@ -148,70 +192,124 @@ class SecurityManager
             const PropertyPolicy& reader_properties,
             EndpointSecurityAttributes& security_attributes);
 
+    /**
+     * Returns a handle to the identity token used in handshake
```

Comment:
I think this is misleading since the actual return type is bool. Maybe use something like "Retrieves a handle". Same comment for the rest of the instances here.

Reply:
done